### PR TITLE
Special scream for loudnigras wearing shorts [DNM]  [WIP] 

### PR DESCRIPTION
**Pull request by ghost**
[Guidelines]: # (Be sure that your PR follows our guidelines, such as modularization and comment standards. You can read more about the subject here: https://github.com/HippieStation/HippieStation/blob/master/hippiestation/README.md )

## Changelog
:cl:
add: Added new scream for people wearing (basketball) shorts
soundadd: added loudnigra1.ogg, loudnigra2.ogg, loudnigra3.ogg
/:cl:

## About The Pull Request
https://cdn.discordapp.com/attachments/321674575245082626/638368834494922762/loudnigra3.ogg
https://cdn.discordapp.com/attachments/321674575245082626/638368835837100044/loudnigra1.ogg
https://cdn.discordapp.com/attachments/321674575245082626/638368837724667928/loudnigra2.ogg
## Why It's Good For The Game
good scream for nibbs

this does not work yet, I put this up to maybe get some help

### CODE REVIEW

**Comment by karmaisblackandbluepilled** [comment]
+1 mashallah

File: hippiestation/code/modules/mob/living/emote.dm [hunk 1]
```diff
@@ -31,6 +31,8 @@
 			return 'hippiestation/sound/voice/cat.ogg'
 		if(user.gender == FEMALE)
 			return pick('hippiestation/sound/voice/scream_f1.ogg', 'hippiestation/sound/voice/scream_f2.ogg')
+		if((H.skin_tone == "african2") || (H.skin_tone == "african1"))
```
Comment (ghost) [inline hippiestation/code/modules/mob/living/emote.dm]: i don't think i can do just mob/living/user aka user here as with the rest, does not recognize the skin_tone var, but it makes no sound like this

**Comment by karmaisblackandbluepilled** [comment]
king, if it doesn't work out, make basketball shorts have that scream

**Comment by ghost** [comment]
followed your advice karma, thanks

**Comment by karmaisblackandbluepilled** [comment]
i'm not karma, thankfully

**Comment by Dystarra** [comment]
+1

No reason not to, all the reason to. 

Can't wait for more fools to go unconscious from scream spamming.

**Comment by Manly-Dwarf** [comment]
-1 Enough alternate screams.
Call me a fun hater but there are too many items right now that change your scream sound. It used to be just the cosbysweater, and people would beat each other to death over that damn thing. It was funny as hell.
Now seeing a guy running around with some obnoxious scream sound is par for the course. I'm kinda sick of it. There's nothing special about it any more, the joke has been beaten to death and now this is all people can come up with for it. You won't see a guy getting into a fight over a Ushanka because he wants to scream "cyka blyat", since there are millions of ways you can get a Ushanka or some other scream-altering item. Two of them you can get roundstart just by selecting them from the Loadout, albeit one of those items you can _only_ get from the Loadout or mugging someone who has it.

> No reason not to, all the reason to.
Can't wait for more fools to go unconscious from scream spamming.

Have some restraint or else you're going to bulldoze anything left that's funny or special in the game. No reason for people to scour maint for the legendary sweater, no interactions that come from it, everyone has their scream sound now. It's a little thing, but it adds up in what makes rounds less interesting in the long run.
stop the bad memes

**Comment by ghost** [comment]
i agree

**Pull request closed without merging**